fetcher/fetcher.go

```diff
@@ -147,7 +147,7 @@ func (bot *fetcherImpl) followLoop(ctx context.Context) error {
 	aclient := bot.Algod()
 	for {
 		for retries := 0; retries < 3; retries++ {
-			_, err = aclient.StatusAfterBlock(bot.nextRound).Do(ctx)
+			_, err = aclient.StatusAfterBlock(bot.nextRound - 1).Do(ctx)
```

Comment:
nit: is there no bot.currentRound?  bot.nextRound - 1 is harder to read.

Reply:
Unfortunately, there is not. I'll add a comment to explain this.